plugins/search-backend-node/src/LunrSearchEngine.ts

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
I wonder if we should stash this under `src/engines/LunrSearchEngine.ts`, in anticipation of other supported search engines in the future?

Not sure if we necessarily want a huge proliferation of backend/node plugins that exist just to support different tech (e.g. `search-backend-elastic-node`, `search-backend-solr-node`, `search-backend-algolia-node`, etc)